backend/infrastructures/apps/extract/adapters/units_of_work.py

```diff
@@ -0,0 +1,18 @@
+from modules.extract.domain.ports import units_of_work as domain_units_of_work
+
+from ...common.adapters import units_of_work
+from .repositories import DjangoFileDomainRepository
+
+
+class DjangoFileUnitOfWork(
```

Comment:
In other point, we discussed that we want to have kind of history for extract. 
Shouldn't we call it `DjangoExtarctUnitOfWork`?

Reply:
Also the same question about repository.

Reply:
This repository and unit of work are dedicated for only saving a file.  
can we use the same unit of work for both saving a file and history?

btw  I am preparing history changes on different branch 3/feature/extract_history